packages/web-components/fast-foundation/src/tabs/tabs.ts

```diff
@@ -242,11 +239,11 @@ export class Tabs extends FASTElement {
             switch (keyCode) {
                 case keyCodeArrowLeft:
                     event.preventDefault();
-                    this.adjust(this.direction === Direction.rtl ? 1 : -1);
+                    this.adjust(1);
```

Comment:
This is backwards I believe. Arrow Key Left should adjust -1.

Reply:
your right! ;)